Makefile

```diff
@@ -287,6 +287,7 @@ $(BUILD_DIR)/macos-amd64/crc-macos-amd64.pkg: packagedir
 
 $(BUILD_DIR)/macos-amd64/crc-macos-installer.tar: packagedir
 	tar -cvf $(BUILD_DIR)/macos-amd64/crc-macos-installer.tar ./packaging
+	pushd $(BUILD_DIR)/macos-amd64 && sha256sum crc-macos-installer.tar > crc-macos-installer.tar.sha256sum && popd
```

Comment:
`sha256sum crc-macos-installer.tar > $(BUILD_DIR)/macos-amd64/crc-macos-installer.tar.sha256sum` ?

Reply:
`crc-macos-installer.tar` file not on the top level dir so we do need to cd to that directory and execute right?

Reply:
@cfergeau If I use that then following will happen which means if we need to verify the checksum either user manually have to cat the file and check with existing or need to create same directory structure and put the target file there.
```
$ sha256sum /tmp/test/bar.txt > /tmp/test/bar.txt.sha256sum
$ cat /tmp/test/bar.txt.sha256sum 
a254679f30b349b670a5098d98e2b47f4a09af80291a81e4eb148e392a52ba82  /tmp/bar.txt
$ mv /tmp/test/* /tmp
$ rm -fr /tmp/test
$ $ sha256sum -c /tmp/bar.txt.sha256sum 
sha256sum: /tmp/test/bar.txt: No such file or directory
/tmp/test/bar.txt: FAILED open or read
sha256sum: WARNING: 1 listed file could not be read
```

Reply:
Ah yes, did not think this would end up adding relative paths to the sha256sum file :)